pegasus/sites.v3/code.org/public/playlab.haml

```diff
@@ -26,17 +26,17 @@ social:
 .playlab-container
   %h1= I18n.t(:playlab_choose_theme)
   %div.image-collection
-    %a.while-link{href: "https://studio.code.org/s/playlab/stage/1/puzzle/1"}
+    %a.white-link{href: "/api/hour/begin/playlab"}
       %div.choice
         %div.choice-container
           %div.choice-label.purple= I18n.t(:playlab_classic_title)
           %img.character-img{src: "/images/playlab_original3.png"}
-    %a{href: "https://studio.code.org/s/gumball/stage/1/puzzle/1"}
+    %a.white-link{href: "/api/hour/begin/gumball"}
       %div.choice
         %div.choice-container
           %div.choice-label.blue= I18n.t(:playlab_gumball_title)
           %img.character-img{src: "/images/playlab_gumball.png"}
-    %a{}
+    %a.white-link{}
       %div.choice
         %div.choice-container
           %div.choice-label.green= I18n.t(:playlab_iceage_title)
```

Comment:
does this say "coming soon"?  if so, we should use a different ID, and also have a string already in i18n being translated ahead of when it's going to start being shown.